Source/WebKit/UIProcess/API/Swift/View+WebViewModifiers.swift

```diff
@@ -63,6 +65,27 @@ extension View {
     public func webViewAllowsElementFullscreen(_ value: Bool = true) -> some View {
         environment(\.webViewAllowsElementFullscreen, value)
     }
+
+    @_spi(Private)
+    public func webViewFindNavigator(isPresented: Binding<Bool>) -> some View {
+        environment(\.webViewFindContext, .init(isPresented: isPresented))
+    }
+
+    @_spi(Private)
+    public func webViewFindDisabled(_ isDisabled: Bool = true) -> some View {
+        transformEnvironment(\.webViewFindContext) { $0.canFind = !isDisabled }
+    }
+
+    @_spi(Private)
+    public func webViewReplaceDisabled(_ isDisabled: Bool = true) -> some View {
+        transformEnvironment(\.webViewFindContext) { $0.canReplace = !isDisabled }
```

Comment:
I'd expect `canReplace` to key off of edibility, as well. Should that be reflected here?

Reply:
it's reflected here https://github.com/WebKit/WebKit/pull/37988/files#diff-c71e2a8d8a171cf1c377fcb17a2d78f09274e84b73e2453fda4f5ed689fd2decR49 ; if the page is not editable, replacement will never be available